wiki_service/core/settings.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+import logging
+from loguru import logger
+from pydantic import BaseSettings
+from typing import Tuple
+from starlette.datastructures import CommaSeparatedStrings
+
+from wiki_service.core.logging import InterceptHandler
+
+
+class Settings(BaseSettings):
+    title: str = 'Kopfsachen Wiki'
+    debug: bool = False
+    allowedHost: CommaSeparatedStrings = CommaSeparatedStrings(os.getenv('ALLOWED_HOSTS', ''))
+
+    api_prefix: str = ''
+
+    mongo_user: str = os.getenv('MONGODB_USER', "admin")
```

Comment:
Be aware that in Kubernetes you won't get all of these values "for free" unless you hardcode them.
What you can get is:
- full connection string with host, username, password and database
- password
- username